Frontend/Models/TextMessage.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Frontend.Models
```

Comment:
Should probably reference the original textmessage class from backend, should  be in Backend.Models